src/main/java/no/sikt/sws/IndexHandler.java

```diff
@@ -36,7 +36,9 @@ protected ProxyResponse<String> processProxyInput(
 
         var resourceIdentifier = request.getPathParameter(RESOURCE_IDENTIFIER);
 
-        if (resourceIdentifier.startsWith("_") || !resourceIdentifier.matches(ALLOWED_INPUT)) {
+        if (!"_alias".equals(resourceIdentifier)
+            &&  resourceIdentifier.startsWith("_")
+            || !resourceIdentifier.matches(ALLOWED_INPUT)) {
```

Comment:
This starts getting hard to read. Should be extracted to a method. We also shouldent use ! and && as its hard to follow.
Would be easier if it was like:
shouldThrowExcaption = reason1 || reason2 || reason3